internal/commands/report/report.go

```diff
@@ -0,0 +1,120 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package report
+
+import (
+	"github.com/apache/skywalking-cli/internal/commands/interceptor"
+	"github.com/apache/skywalking-cli/internal/logger"
+	"github.com/apache/skywalking-cli/internal/model"
+	"github.com/apache/skywalking-cli/pkg/display"
+	"github.com/apache/skywalking-cli/pkg/display/displayable"
+	"github.com/apache/skywalking-cli/pkg/grpc"
+
+	"github.com/urfave/cli"
+
+	event "skywalking/network/event/v3"
+)
+
+var Command = cli.Command{
+	Name:      "report",
+	Aliases:   []string{"r"},
+	Usage:     "Report an event to OAP server via gRPC",
+	ArgsUsage: "[parameters...]",
+	Flags: []cli.Flag{
+		cli.StringFlag{
+			Name:     "grpcAddr",
```

Comment:
Because we have multiple commands using this, let's add this as a top level flag, and make it possible to be specified by the `~/.skywalking.yaml`, just like [`base-url`](https://github.com/apache/skywalking-cli/blob/160f2bd2ebb8395d25b728ddaae4dc9995742f79/cmd/swctl/main.go#L61)